### PR TITLE
Responsive UI and toast

### PR DESCRIPTION
**Pull request by Jia-A**
In this PR, I have added responsiveness and toast for Dot Note app.

### CODE REVIEW

File: src/note-components/navbar/navbar.jsx [hunk 1]
```diff
@@ -1,25 +1,92 @@
-import { Link } from "react-router-dom";
+import { Link, NavLink , useNavigate } from "react-router-dom";
 import "./navbar.css";
+import "../notepage/note.css"
 import { useTheme } from "../../Context/theme-context";
+import { useState } from "react";
+import { useAuth } from "../../Context/authorization-context";
+import { useNote } from "../../Context/note-context";
+import { useFilter } from "../../Context/filter-context";
```
Comment (maitrakhatri) [inline src/note-components/navbar/navbar.jsx:8]: us barrel exports

**Review by maitrakhatri — APPROVED** [review]
You're repeating the change them button in 3 pages, you can make it an independent component and use it instead of repeating the code. Also use barrel exports for components and contexts. 
Rest looks good, clean code, easy to read

File: src/note-components/navbar/navbar.jsx (hunk 1 above)
Comment (Jia-A) [reply src/note-components/navbar/navbar.jsx:8]: Okay will do that....

**Review by Varu98 — APPROVED** [review]
Clean Code and Modular File Structure, Nice!

**Pull request merged**